src/main/java/com/twilio/oai/StringHelper.java

```diff
@@ -48,4 +50,27 @@ private String convertFirstChar(final String inputWord, final UnaryOperator<Stri
             ? inputWord
             : firstCharFunction.apply(inputWord.substring(0, 1)) + inputWord.substring(1);
     }
+
+    public static String convertUrlToCamelCase(String input) {
+        if (input == null || input.isEmpty()) {
+            return input;
+        }
+
+        Pattern pattern = Pattern.compile("\\{([a-z0-9_]+)\\}");
```

Comment:
If this is a direct toCamelCase conversion, can we use a library to do it? [example](https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/CaseUtils.html#toCamelCase(java.lang.String,boolean,char...))
If its not what format does it return ?

Reply:
This function takes input of format `/v1/Services/{account_sid}/List` to `/v1/Services/{AccountSid}/List`
The CaseUtils.toCamelCase() does not have this exact usecase fulfilled.